### PR TITLE
Simplify permissions in flows app

### PR DESCRIPTION
**Pull request by rowanseymour**
* Views required by editor should require the same permission as the editor
 * Views required by results page should require the same permission as the results page
 * etc

### CODE REVIEW

File: templates/flows/flow_list.html [hunk 1]
```diff
@@ -2,9 +2,11 @@
 {% load smartmin sms temba compress i18n humanize %}
 
 {% block content %}
-  {% if org_perms.flows.flow_update %}
```
Comment (rowanseymour) [inline templates/flows/flow_list.html:5]: this is wrong...

File: temba/flows/views.py [hunk 2]
```diff
@@ -953,14 +958,13 @@ def build_content_menu(self, menu):
 
             # limit PO export/import to non-archived flows since mailroom doesn't know about archived flows
             if not obj.is_archived:
-                if self.has_org_perm("flows.flow_export_translation"):
-                    menu.add_modax(
-                        _("Export Translation"),
-                        "export-translation",
-                        reverse("flows.flow_export_translation", args=[obj.id]),
-                    )
+                menu.add_modax(
```
Comment (norkans7) [inline temba/flows/views.py:961]: I guess we check editor permission here

Comment (rowanseymour) [reply temba/flows/views.py:961]: we don't need to - this is inside the Editor view

File: temba/flows/tests.py [hunk 3]
```diff
@@ -3164,8 +3164,8 @@ def test_export_and_download_translation(self):
         flow = self.get_flow("favorites")
         export_url = reverse("flows.flow_export_translation", args=[flow.id])
 
-        self.assertRequestDisallowed(export_url, [None, self.user, self.agent, self.admin2])
```
Comment (rowanseymour) [inline temba/flows/tests.py:3167]: viewers should be able to export translations.. that's just reading

**Review by norkans7 — APPROVED** [review]

**Pull request merged**